components/popup/test/popup.test.js

```diff
@@ -4,9 +4,9 @@ import 'chai-dom';
 import {
 	waitNextTask,
 	textToDomToParent,
-	isolatedElementsCreation,
+	isolatedElementsCreation, waitInterval, noop,
```

Comment:
is _waitInterval_ needed?

Reply:
Maybe not. Don't we have a linting rule that should fail if unused variable?
Anyway - if it doesn't appear in the code, it can be removed.